GeoLib/DuplicateGeometry.h

```diff
@@ -0,0 +1,69 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef DUPLICATEGEOMETRY_H_
+#define DUPLICATEGEOMETRY_H_
+
+#include <string>
+
+#include "GeoLib/GEOObjects.h"
```

Comment:
Argh .... I accidently wrote a forward declaration *and* a normal declaration in the header file. Using *only* the forward declaration the compile errors are back and it really doesn't seem to work.